testing/testing.gni

```diff
@@ -124,26 +124,15 @@ template("dart_snapshot_aot") {
       invoker.dart_kernel,
     ]
 
-    vm_snapshot_data = "$target_gen_dir/assets/vm_snapshot_data"
-    vm_snapshot_instr = "$target_gen_dir/assets/vm_snapshot_instr"
-    isolate_snapshot_data = "$target_gen_dir/assets/isolate_snapshot_data"
-    isolate_snapshot_instr = "$target_gen_dir/assets/isolate_snapshot_instr"
+    elf_object = "$target_gen_dir/assets/binary.so"
 
-    outputs = [
-      vm_snapshot_data,
-      vm_snapshot_instr,
-      isolate_snapshot_data,
-      isolate_snapshot_instr,
-    ]
+    outputs = [elf_object]
 
     args = [
       "--causal_async_stacks",
       "--deterministic",
-      "--snapshot_kind=app-aot-blobs",
-      "--vm_snapshot_data=" + rebase_path(vm_snapshot_data),
-      "--vm_snapshot_instructions=" + rebase_path(vm_snapshot_instr),
-      "--isolate_snapshot_data=" + rebase_path(isolate_snapshot_data),
-      "--isolate_snapshot_instructions=" + rebase_path(isolate_snapshot_instr),
+      "--snapshot_kind=app-aot-elf",
```

Comment:
In the tests, can we keep generating both snapshots for now? In the `EmbedderConfigBuilder`, we can switch to using  the `app-aot-elf`s but we should also add at least a single test that explicitly checks loading from the old blobs till embedders migrate or support for the same is dropped from `gen_snapshot`.

Reply:
I've changed the PR to not change embedder.h, so the existing fields of FlutterProjectArgs are still being tested.